meinberlin/static/embed.js

```diff
@@ -28,7 +28,7 @@ $(document).ready(function () {
     }
   })
 
-  $(document).on('submit', function (event) {
+  $(document).on('submit', 'form[action]', function (event) {
```

Comment:
Out of interest (because I forgot): What are the cases where a submit event is fired from something that does not match `form[action]`? Why do we not want to handle these cases?